packages/svg-icons/src-cs/Telerik.SvgIcons/Models/SvgIconBase.cs

```diff
@@ -7,5 +7,20 @@ public class SvgIconBase : ISvgIcon
         public string Content { get; set; }
 
         public string ViewBox { get; set; }
+
+        public override bool Equals(object obj)
+        {
+            return Equals(obj as SvgIconBase);
+        }
+
+        public bool Equals(SvgIconBase other)
+        {
+            return other != null && Content == other.Content;
+        }
+
+        public override int GetHashCode()
+        {
+            return Content.GetHashCode();
+        }
```

Comment:
We discussed this with @radkostanev and the methods will be updated to include all 3 parameters - Name, Content and ViewBox and check for their equality with the other object.

Reply:
Addressed in b33ff8f